app/models/cms/builtin.rb

```diff
@@ -135,20 +135,17 @@ class Page < CMS::Builtin
     def to_xml(options = {})
       xml = options[:builder] || Nokogiri::XML::Builder.new
 
-      xml.builtin_page do |x|
+      xml.__send__(CMS::XML::CLASS_TAG.call(self.class)) do |x|
```

Comment:
I think you can do this without a lookup table like this:
```
self.class.name[5..-1].parameterize(separator: "_")
```

Reply:
I think I prefer to leave the lookup table, because it works as a single source of truth for tags reference and shouldn't be a difference in performance, right?

Reply:
To me a transformation method would be a better source of truth because it immediately shows that there is a convention without the need to figure this out in the lookup table.

Ideally we would have a method `self.class.xml_tag` or if that is probably also valid for JSON, maybe `::response_tag` or `::data_tag` or something better. And I assume this method can be defined in some parent class.

Reply:
I committed the last idea: a new `self.class.data_tag` attribute